snaps/reference/exports.md

```diff
@@ -270,3 +270,143 @@ module.exports.onCronjob = async ({ request }) => {
 ```
 
 <!--/tabs-->
+
+## onInstall
+
+To run an action on installation, a Snap must export `onInstall`.
+This method is called after the Snap is installed successfully. 
+
+:::note
+For MetaMask to call the Snap's `onInstall` method, you must request the
+[`endowment:lifecycle-hooks`](permissions.md#endowmentlifecycle-hooks) permission.
+:::
+
+#### Parameters
+
+None.
+
+#### Returns
+
+A promise containing the return of the implemented method.
```

Comment:
```suggestion
```

I don't think mentioning a return value makes sense here, similar to cronjobs.

Reply:
I think it's worthwhile to mention that it returns a promise